telos-backend/demo.js

```diff
@@ -1,15 +1,39 @@
 const TodoTask = require('./src/models/todo_task');
 
+// Check if there are any TodoTask documents
 TodoTask.find().countDocuments((err, count) => {
+  // If there are none
   if (count === 0) {
-    console.log('Empty');
+    // create new TodoTask document
     const entry = new TodoTask({
       name: 'Lorem',
       createdDate: '2021-01-01',
       dueDate: '2021-01-01',
       completed: false,
     });
+    // save document
+    entry.save();
+  }
+});
 
+const Text = require('./src/models/text');
+
+// look for existing text documents
+Text.find().countDocuments((err, count) => {
```

Comment:
Could do "Creating demo data for Text widget" instead of just saying what the code says.